restcomm/restcomm.interpreter/pom.xml

```diff
@@ -172,5 +172,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
```

Comment:
we should start putting test dependencies in root pom, but this is not particular to this PR

Reply:
Agreed. I must admit took this from another PR 😬. I'll move to root then, ok?

Reply:
i did the same for another PR, but im seeing we are all adding test dependencies per subproject, and even worse duplciating the version tag. 
maybe we could add all test dependecies at root level to ensure version consistency, or at least use dependencymanagement to fix version at root level.

Reply:
Moved to root using `dependencymanagement `.